frontend/src/state-providers/fonts.ts

```diff
@@ -46,11 +46,11 @@ export function createFontsState(editor: Editor) {
 
 		const response = await fetch(fontFileUrl);
 		const responseBuffer = await response.arrayBuffer();
```

Comment:
For consistency with the one below, can you combine these into a single line for me? Thanks.